### PR TITLE
Fix: Customer list trap focus does not work properly - CXSPA 1887 

### PR DESCRIPTION
**Pull request by bullfroggy-sap**
Add config option for TrapFocusConfig to specify when up and down keyboard events should be ignored
Implement ignoreArrows config in trap-focus directive
Use ignoreArrows config in customer-list component

### CODE REVIEW

**Review by ercultimate — DISMISSED** [review]

File: feature-libs/asm/components/customer-list/customer-list.component.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export class CustomerListComponent implements OnInit, OnDestroy {
     trap: true,
     block: true,
     autofocus: 'customer-list-selector',
+    ignoreArrows: true,
```
Comment (justinlee01) [inline feature-libs/asm/components/customer-list/customer-list.component.ts]: Can this be changed to `ignoreArrowKeys`?
Or, more precisely, `ignoreUpDownArrowKeys`?

File: projects/storefrontlib/layout/a11y/keyboard-focus/keyboard-focus.model.ts [hunk 2]
```diff
@@ -113,6 +113,11 @@ export type TrapFocusType =
 export interface TrapFocusConfig extends TabFocusConfig {
   /** traps the focus */
   trap?: TrapFocusType;
+
+  /**
+   * Indicates that up and down arrow keyboard events should be ignored
```
Comment (justinlee01) [inline projects/storefrontlib/layout/a11y/keyboard-focus/keyboard-focus.model.ts]:
```suggestion
   * Indicates that up and down arrow keyboard events should be ignored. Works only if 'trap' is truthy.
```

**Review by justinlee01 — CHANGES_REQUESTED** [review]

File: feature-libs/asm/components/customer-list/customer-list.component.ts (hunk 1 above)
Comment (bullfroggy-sap) [reply feature-libs/asm/components/customer-list/customer-list.component.ts]: yea, ok

File: projects/storefrontlib/layout/a11y/keyboard-focus/keyboard-focus.model.ts (hunk 2 above)
Comment (bullfroggy-sap) [reply projects/storefrontlib/layout/a11y/keyboard-focus/keyboard-focus.model.ts]: Ok

**Review by justinlee01 — APPROVED** [review]

**Review by ercultimate — APPROVED** [review]

**Review by plabadie — APPROVED** [review]

**Pull request merged**